### ISSUE TITLE
Some of the Error responses on POST, PUT requests are missing in web api document

### ISSUE THREAD

**Issue by kanta1207** [genesis-tech-tribe/nishiki-documents#145]
## Why is this enhancement important?
Some of the Error responses on POST, PUT requests are missing in web api document.
For example, on updating or deleting container, there probably should be error responses when `containerId' is incorrect.

It would be great if we can talk through this when we can!

**Comment by hitohata** [comment]
Thank you for pointing it out.
Do you need the all error response? I mean the error types, especially those that come from input error, get a lot of patterns. Or, just add 400?

Let's say you are going to call GET /container/:containerId. This endpoint can return the following errors except for the forbidden error:

* incorrect container ID
* incorrect user ID

**Comment by kanta1207** [comment]
@hitohata 
Sorry for the late reply...
After the verbal conversation a min ago, I came to understand `404` is not appropriate for the cases where groups or containers weren't found with the given id, since the resource API is certainly existing.
 And, I also got it that `400` is omitted in the doc, so my concern ↓ is solved!
 >on updating or deleting container, there probably should be error responses when `containerId' is incorrect.
 
 Let me talk to you again or close this issue after the frontend meeting, if there's any additional concern coming up.
